e2e/openssl/openssl_test.go

```diff
@@ -78,9 +80,43 @@ func TestOpenSSL(t *testing.T) {
 		require.NoError(err)
 		require.NotEmpty(coordinatorPods, "pod not found: %s/%s", ct.Namespace, "coordinator")
 
+		// deploy an additional port forwarder for metrics and probes
+		stableNetworkID := fmt.Sprintf("%s.coordinator.%s.svc.cluster.local", coordinatorPods[0].Name, ct.Namespace)
+		t.Logf("Constructed stable network ID %q", stableNetworkID)
+		additionalForwarder := kuberesource.
+			PortForwarder("coordinator-metrics", ct.Namespace).
+			WithListenPorts([]int32{9102}).
+			WithForwardTarget(stableNetworkID)
+
+		patchedForwarder, err := kuberesource.ResourcesToUnstructured(kuberesource.PatchImages([]any{additionalForwarder.PodApplyConfiguration}, ct.ImageReplacements))
+		require.NoError(err)
+		require.NoError(ct.Kubeclient.Apply(ctx, patchedForwarder...))
+
+		t.Cleanup(func() {
+			_ = ct.Kubeclient.Client.CoreV1().Pods(ct.Namespace).Delete(context.Background(), "port-forwarder-coordinator-metrics", metav1.DeleteOptions{})
+		})
+
 		argv := []string{"/bin/sh", "-c", "curl --fail " + net.JoinHostPort(coordinatorPods[0].Status.PodIP, "9102") + "/metrics"}
 		_, stderr, err := ct.Kubeclient.Exec(ctx, ct.Namespace, frontendPods[0].Name, argv)
 		require.NoError(err, "stderr: %q", stderr)
+
+		for _, endpoint := range []string{"/probe/startup", "/probe/liveness", "/probe/readiness"} {
+			require.NoError(ct.Kubeclient.WithForwardedPort(ctx, ct.Namespace, "port-forwarder-coordinator-metrics", "9102", func(addr string) error {
+				req, err := http.NewRequestWithContext(ctx, http.MethodGet, "http://"+addr+endpoint, nil)
+				if err != nil {
+					return err
+				}
+				resp, err := http.DefaultClient.Do(req)
+				if err != nil {
+					return err
+				}
+				defer resp.Body.Close()
+				if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusServiceUnavailable {
```

Comment:
```suggestion
				if resp.StatusCode != http.StatusOK {
```
At this point in the test, the Coordinator should be ready.